### PR TITLE
[feature] xmldb:find-last-modified-until

### PR DESCRIPTION
**Pull request by olvidalo**
Adds `xmldb:find-last-modified-until` analogous to the already existing `xmldb:find-last-modified-since`. 

Also fixes the documentation for the return value of `find-last-modified-since` and adds tests for both functions.

### CODE REVIEW

File: test/src/xquery/xmldb/suite.xql [hunk 1]
```diff
@@ -2,6 +2,8 @@ xquery version "1.0";
 
 import module namespace test="http://exist-db.org/xquery/xqsuite" 
 at "resource:org/exist/xquery/lib/xqsuite/xqsuite.xql";
-import module namespace t="http://exist-db.org/testsuite/permissions" at "permission-tests.xql";
 
-test:suite(util:list-functions("http://exist-db.org/testsuite/permissions"))
+test:suite((
+    (: inspect:module-functions(xs:anyURI("permission-tests.xql")),:)
```
Comment (adamretter) [inline test/src/xquery/xmldb/suite.xql]: Just wondering why the permission-tests are commented out now?

File: src/org/exist/xquery/functions/xmldb/FindLastModified.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
+package org.exist.xquery.functions.xmldb;
```
Comment (adamretter) [inline src/org/exist/xquery/functions/xmldb/FindLastModified.java]: Could you add the standard copyright header?

**Comment by adamretter** [comment]
@olvidalo Looks good. Just a couple of comments...

File: test/src/xquery/xmldb/suite.xql (hunk 1 above)
Comment (olvidalo) [inline test/src/xquery/xmldb/suite.xql]: Ooops! I've amended the testsuite commit.

**Comment by adamretter** [comment]
@olvidalo Thanks for your contribution :-)

**Comment by olvidalo** [comment]
Wow that was quick. Thanks for merging!

**Pull request merged**